doc/stdenv/meta.chapter.md

```diff
@@ -47,6 +47,10 @@ Release branch. Used to specify that a package is not going to receive updates t
 
 The package’s homepage. Example: `https://www.gnu.org/software/hello/manual/`
 
+### `repository` {#var-meta-repository}
+
+A webpage where the package's source code can be viewed.  `https` links are preferred if available.  Automatically set to a default value if the package uses a `fetchFrom*` fetcher for its `src`. Example: `https://github.com/forthy42/gforth`
```

Comment:
```suggestion
A webpage (or a list of webpages) where the package's source code can be viewed.  `https` links are preferred if available.  Automatically set to a default value if the package uses a `fetchFrom*` fetcher for its `src`. Example: `https://github.com/forthy42/gforth`
```

P.S.: Again, this is not the official upstream source for Gforth. 

And, since the xzdoor incident (that happened on its github mirror), we should not promote mirrors.